participant/Makefile

```diff
@@ -0,0 +1,9 @@
+.PHONY : \
```

Comment:
phony?

Reply:
For this: https://stackoverflow.com/questions/2145590/what-is-the-purpose-of-phony-in-a-makefile